pyquil/latex/_diagram.py

```diff
@@ -322,23 +340,23 @@ class DiagramBuilder:
     recursive methods.
     """
 
-    def __init__(self, circuit, settings):
+    def __init__(self, circuit: Program, settings: DiagramSettings):
         self.circuit = circuit
         self.settings = settings
         # instructions currently being processed
-        self.working_instructions = None
+        self.working_instructions: Optional[List[AbstractInstruction]] = None
         # index into working instructions. we maintain the invariant that
         # working_instructions[0:index] has been processed, with the diagram
         # updated accordingly
         self.index = 0
         # partially constructed diagram
-        self.diagram = None
+        self.diagram: Optional[DiagramState] = None
 
-    def build(self):
+    def build(self) -> DiagramState:
         """
         Actually build the diagram.
         """
-        qubits = self.circuit.get_qubits()
+        qubits = cast(Set[int], self.circuit.get_qubits())
```

Comment:
```suggestion
        qubits = cast(Set[int], self.circuit.get_qubits(indices=True))
```

Also, maybe this should `assert` that there aren't any `QubitPlaceholder` in the returned qubits? I assume the whole latex machinery won't work with `QubitPlaceholder`s anyway, but otherwise the `cast` is a little white lie.

Reply:
isn't `indices=True` the default?

Reply:
it is the default, but i'll make it explicit anyways

Reply:
oh snap